src/test/java/org/broadinstitute/hellbender/tools/walkers/vqsr/CNNScoreVariantsIntegrationTest.java

```diff
@@ -242,10 +246,24 @@ public void testInferenceArchitectureAndWeights2d() throws IOException {
                 .addArgument("tensor-type", tt.name())
                 .addArgument(StandardArgumentDefinitions.ADD_OUTPUT_VCF_COMMANDLINE, "false");
 
+        runCommandLine(argsBuilder);
+        assertInfoFieldsAreClose(tempVcf, expectedVcf, GATKVCFConstants.CNN_2D_KEY);
+    }
 
-        final IntegrationTestSpec spec = new IntegrationTestSpec(argsBuilder.toString(),
-                Arrays.asList(largeFileTestDir + "VQSR/expected/cnn_2d_chr20_subset_expected.vcf"));
-        spec.executeTest("testInference2d", this);
+    private void assertInfoFieldsAreClose(File vcf1, File vcf2, String infoKey){
+        Iterator<VariantContext> vi1 = VariantContextTestUtils.streamVcf(vcf1).collect(Collectors.toList()).iterator();
+        Iterator<VariantContext> vi2 = VariantContextTestUtils.streamVcf(vcf2).collect(Collectors.toList()).iterator();
+        while (vi1.hasNext() && vi2.hasNext()) {
+            VariantContext vc1 = vi1.next();
+            VariantContext vc2 = vi2.next();
+            double v1Score = vc1.getAttributeAsDouble(infoKey, 0.0); // Different defaults trigger failures on missing scores
+            double v2Score = vc2.getAttributeAsDouble(infoKey, EPSILON+1.0);
+            double diff = Math.abs(v1Score-v2Score);
+            Assert.assertTrue(diff < EPSILON);
```

Comment:
This should still ensure we're not dropping anything else:

```suggestion
            Assert.assertTrue(diff < EPSILON);
            VariantContextTestUtils.assertVariantContextsAreEqual (actual, expected, Collections.singletonList(infoKey))
```

Reply:
done